IdokladSdk.IntegrationTests/Core/Extensions/ApiResultExtensions.cs

```diff
@@ -67,5 +48,20 @@ public static async Task<IEnumerable<T>> AssertResult<T>(this Task<ApiBatchResul
 
             return result.Results.Select(x => x.Data);
         }
+
+        /// <summary>
+        /// Assert result.
+        /// </summary>
+        /// <param name="result">Result.</param>
+        /// <typeparam name="T">T.</typeparam>
+        /// <returns>Method assert result.</returns>
+        private static T AssertResult<T>(this ApiResult<T> result)
```

Comment:
Tady ta metoda se ještě někde volá, když jsi mazal závorky z těch testů, aby byla použita ta asynchronní verze metody?
public static async Task<T> AssertResult<T>(this Task<ApiResult<T>> result)

Reply:
áno, používa sa v tých verejných async metódach